grid/cell.py

```diff
@@ -34,19 +33,28 @@ class Cell(NamedTuple):
     the panels.
     """
 
-    position: "Position"
+    coordinate: Coordinate
     orientation: Orientation
     addresses: List["Address"]
     geom: Geometry
+    real: bool
```

Comment:
Could be an enum, cell in a panel, cell in the blank space of the face, and cell off the edge of the face.

Reply:
> off the edge of the face

I put back the `coordinate not in self.geom` check, so if you try to access a cell that's not even defined in the whole `Face` geometry, it's undefined. `Cell`s with `real=false` will only be emitted for the gaps between panels